src/routes/events/+page.svelte

```diff
@@ -2,8 +2,7 @@
 	import Societies from '$lib/components/Societies/index.svelte';
 	import EventListElement from '$lib/components/EventListElement/index.svelte';
 	import Seo from '$lib/components/Seo.svelte';
-	import type { PageData } from './$types';
-	export let data: PageData;
```

Comment:
If `data` was used inside the `<script lang="ts">` I would argue that removing the type is only applicable if you use the Svelte(Kit) Language Server, and be extra explicit won't harm anyone.

But, as `data` is only used inside the markup, where only the Svelte Language server can suggest completion on variable inside `{}` and `{# ...}` and `{@ ...}` I won't be overly preventive